src/main/java/mate/academy/spring/model/User.java

```diff
@@ -16,6 +18,8 @@ public class User {
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany
+    private Set<Role> roles;
```

Comment:
Consider adding a @JoinTable to specify the table and columns used for storing the relationship between User and Role entities.